src/db/migrations/20201022155502-create-product.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Products', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.DataTypes.INTEGER
+      },
+      category: {
+        type: Sequelize.DataTypes.STRING
+      },
+      productname: {
```

Comment:
Perhaps "productname" should just be "name" according to the naming conventions?

Reply:
Agreed